l10n_it_delivery_note/controllers/portal.py

```diff
@@ -0,0 +1,109 @@
+from odoo import _
+from odoo.exceptions import AccessError, MissingError
+from odoo.http import request, route
+
+from odoo.addons.portal.controllers.portal import CustomerPortal, pager as portal_pager
+
+
+class DNCustomerPortal(CustomerPortal):
+    def _get_delivery_note_domain(self):
+        return [("state", "in", ["confirm", "invoiced", "done"])]
```

Comment:
è una scelta non visualizzare i ddt in bozza ed annullati?

Reply:
Sì, ma si può discutere

Reply:
Sì, l'ha richiesto @francesco-ooops, ma condivido